message_ix_models/model/water/report.py

```diff
@@ -1374,12 +1396,15 @@ def report(sc: Scenario, reg: str, sdgs: bool = False) -> None:
     report_pd = report_pd[-report_pd.variable.isin(water_hydro_var)]
 
     # add water population
-    pop_sdg6 = pop_water_access(sc, reg, sdgs)
-    report_pd = pd.concat([report_pd, pop_sdg6])
-
-    # add units
-    for index, row in map_agg_pd.iterrows():
-        report_pd.loc[(report_pd.variable == row["names"]), "unit"] = row["unit"]
+    if sdgs != "baseline":
```

Comment:
Ideally we want to report the population with access to water and sanitation with any scenario.
I see in `demand.py` we only generate `all_rates_SSP2.csv` in the SDG case. 
If we change that also for the baseline the file is generated, then we are able to run this function for baseline too, right?
The rates for baseline do exist, and the current formatting was probably just for some project questions, but it would be good to make it work for baseline too, so one can compare.

you could either leave a comment here and we do it in the next PR, or we make the .csv also for baseline. what do you think?